core/domain/translation_services.py

```diff
@@ -288,20 +289,23 @@ def compute_translation_related_changes_upon_revert(
 
 
 def get_languages_with_complete_translation(
-    exploration: exp_domain.Exploration
+    entity: Union[exp_domain.Exploration, study_guide_domain.StudyGuide],
+    entity_type: feconf.TranslatableEntityType
 ) -> List[str]:
-    """Returns a list of language codes in which the exploration translation
-    is 100%.
+    """Returns a list of language codes in which the entity translation is 100%.
+
+    Args:
+        entity: The translatable entity (Exploration, Study Guide, etc.).
+        entity_type: The type of the translatable entity.
 
     Returns:
         list(str). A list of language codes in which the translation for the
-        exploration is complete i.e, 100%.
+        entity is complete i.e, 100%.
     """
-    content_count = exploration.get_content_count()
+    content_count = entity.get_content_count(entity_type)
```

Comment:
Why do we need to pass entity_type when particular entity's get_content_count() method is called? I think this would possibly break the existing system.